x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -4436,6 +4436,7 @@
     "visTypeTimeseries.timeseries.optionsTab.dataLabel": "データ",
     "visTypeTimeseries.timeseries.optionsTab.displayGridLabel": "グリッドを表示",
     "visTypeTimeseries.timeseries.optionsTab.ignoreGlobalFilterLabel": "グローバルフィルターを無視しますか？",
+    "visTypeTimeseries.timeseries.optionsTab.ignoreDaylightTimeLabel": "夏時間を無視しますか？",
```

Comment:
You shouldn't add the new translations. There is a team that finds the new texts and translates them, so no need to change these files

Reply:
Removed